### PR TITLE
(CI): add appveyor for windows build

### PR DESCRIPTION
**Pull request by matklad**
Add [Appveyor](http://www.appveyor.com/) for windows build. 

@alexeykudinkin you should add a webhook to launch the build on commit (not sure about this).  

I don't know if travis + appveyor is the best way to do CI, but it seems ok so far.

### CODE REVIEW

**Comment by Turbo87** [comment]
and now you're the mind reader... ;)

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Rust IDE built using the IntelliJ Platform
 
-[![Build Status](https://img.shields.io/travis/alexeykudinkin/intellij-rust/master.svg)](https://travis-ci.org/alexeykudinkin/intellij-rust) [![Join the chat at https://gitter.im/alexeykudinkin/intellij-rust](https://img.shields.io/badge/Gitter-Join%20Chat-blue.svg)](https://gitter.im/alexeykudinkin/intellij-rust?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
+[![Build Status Travis](https://img.shields.io/travis/alexeykudinkin/intellij-rust/master.svg)](https://travis-ci.org/alexeykudinkin/intellij-rust)
+[![Build Status Appveyor](https://ci.appveyor.com/api/projects/status/r4fc684iuxda17wj/branch/master?svg=true)](https://ci.appveyor.com/project/matklad/intellij-rust/branch/master)
```
Comment (Turbo87) [inline README.md]: you can use [shields.io](http://shields.io/) here too:

[Build Status Appveyor](https://ci.appveyor.com/project/matklad/intellij-rust/branch/master)

```md
[![Build Status Appveyor](https://img.shields.io/appveyor/ci/matklad/intellij-rust/master.svg)](https://ci.appveyor.com/project/matklad/intellij-rust/branch/master)
```

Comment (Turbo87) [inline README.md]: maybe we should add `?label=TravisCI` and `?label=AppVeyor` to overwrite the default `build` label in this case

[Build Status Appveyor](https://ci.appveyor.com/project/matklad/intellij-rust/branch/master)

Comment (matklad) [inline README.md]: Thanks for the `label` tip! 

I didn't use `shieds.io` just to make Travis and AppVeyor badges different :)

**Comment by Turbo87** [comment]
@alexeykudinkin @matklad is there a reason for not merging this? would probably be a good idea to test all of the stuff on Windows too

**Comment by matklad** [comment]
@Turbo87  we are planing to use TeamCity for CI.

**Comment by Turbo87** [comment]
I guess this PR can be closed then, right?

**Pull request closed without merging**